ticker_test.go

```diff
@@ -39,7 +45,7 @@ func TestFakeTickerTick(t *testing.T) {
 		if tick != first {
 			t.Errorf("wrong tick time, got: %v, want: %v", tick, first)
 		}
-	case <-time.After(time.Millisecond):
+	case <-ctx.Done():
```

Comment:
Sorry for the delay. Taking care of this today.

Reply:
I think the context is worth it for a few reasons:
1. When a mistake is made and multiple tests lock up, it is nice to have 1 and only 1 place to modify the timeout (shorten or lengthen it) as desired. Without this, the engineer has to modify each specific test.
2. In general, it is preferable to have this slightly longer, certainly longer than 1 ms. This is because if a machine's scheduler is over taxed, it might not schedule the event in time, and the test runner will report a flake failure rather than a pass. A longer time out protects against this occurring.

Reply:
I came back to check something (and make sure I was convinced of my own position) and I also noticed another reason I believe this to be preferable:

Having a single, somewhat long global time out means we don't have to define stand alone timeouts at each point in the test where we are waiting on a receive. The other options I considered were:

1. Use `time.After(timeout)` rather than `ctx.Done()`. This would be the overall test timeout is N*timeout where N is the number of receives within the test case. That seemed a bit silly.
2. Just define a different duration for each of these receives. That seemed cumbersome, and induced what I felt was unnecessary cognitive overhead: these delays should all be trivial, implicitly making future contributors to reason about the "right" durations for each instance didn't seem to add meaningful value.